src/PojoMap.ts

```diff
@@ -7,6 +7,56 @@ export type PojoMap<T extends PropertyKey, U extends {}> = {
   readonly [idx in T]?: U;
 };
 
+function fromEntries<T extends PropertyKey, U extends {}>(
+  entries: Readonly<Array<readonly [T, U]>>,
+): PojoMap<T, U> {
+  const acc: Partial<Record<T, U>> = {};
+  for(const [key, value] of entries) {
+    acc[key] = value;
+  }
+  return acc;
+}
+
+/**
+ * Check if an value is in a PojoMap.
+ *
+ * @param map A PojoMap
+ * @param value The value to check existence of
```

Comment:
parameter is a key, not a value.
```suggestion
 * @param key The key to check existence of
```